src/hosted-form/hosted-form.ts

```diff
@@ -63,7 +67,14 @@ export default class HostedForm {
         return await this._getFirstField().submitForm(
             this._fields.map(field => field.getType()),
             this._payloadTransformer.transform(payload)
-        );
+        )
+            .catch((error: RequestError) =>
```

Comment:
Sorry, I meant this `catch` block needs to be converted into `async/await` as well.